pkgs/tools/backup/mylvmbackup/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchurl
+, perlPackages
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mylvmbackup";
+  version = "0.16";
+
+  src = fetchurl {
+    url = "${meta.homepage}/${pname}-${version}.tar.gz";
```

Comment:
Please don't use meta.homepage in download URLs as this breaks download links unexpectedly when updating homepages.

Reply:
https://github.com/NixOS/nixpkgs/pull/137033